test/data_dir/data_file3.toml

```diff
@@ -0,0 +1,40 @@
+gid = 75
```

Comment:
Thanks for adding full files for testing, this is going to be very useful to develop further tests.

Reply:
🚀